### PR TITLE
Add fname option to plot_over methods

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Add fname option to plot_over methods

 

### Details

- We can save figure like follwing.
```python
import pyvista as pv
from pyvista import examples

mesh = examples.download_kitchen()

a = [mesh.bounds[0], mesh.bounds[2], mesh.bounds[4]]
b = [mesh.bounds[1], mesh.bounds[3], mesh.bounds[5]]

mesh.plot_over_line(a, b, resolution=100, fname="kitchen.png")
```
kitchen

### CODE REVIEW

**Review by tkoyama010 — COMMENTED** [review]
WFM

File: pyvista/core/filters.py [hunk 1]
```diff
@@ -2127,6 +2127,9 @@ def plot_over_line(dataset, pointa, pointb, resolution=None, scalars=None,
             Tolerance used to compute whether a point in the source is in a
             cell of the input.  If not given, tolerance is automatically generated.
 
+        fname : str
+            The string file name of the save figure.
```
Comment (akaszynski) [inline pyvista/core/filters.py:2131]:
```suggestion
        fname : str, optional
            Save the figure this file name when set.
```

File: pyvista/core/filters.py [hunk 2]
```diff
@@ -2317,6 +2322,9 @@ def plot_over_circular_arc(dataset, pointa, pointb, center,
             in a cell of the input.  If not given, tolerance is
             automatically generated.
 
+        fname : str
+            The string file name of the save figure.
```
Comment (akaszynski) [inline pyvista/core/filters.py:2326]:
```suggestion
        fname : str, optional
            Save the figure this file name when set.
```

File: pyvista/core/filters.py [hunk 3]
```diff
@@ -2426,6 +2436,9 @@ def plot_over_circular_arc_normal(dataset, center, resolution=None, normal=None,
...
```
Comment (akaszynski) [inline pyvista/core/filters.py:2440]:
```suggestion
        fname : str, optional
            Save the figure this file name when set.
```

**Review by akaszynski — APPROVED** [review]
LGTM with suggestions.

**Review by tkoyama010 — COMMENTED** [review]
Thanks

**Pull request merged**